about.css

```diff
@@ -0,0 +1,312 @@
+  padding: 2%;
+}
+
+.email-address {
+  font-family: 'Lato', sans-serif;
+  color: #fff;
+  margin-left: 20%;
+  margin-right: 20%;
+  margin-bottom: 10%;
+  font-weight: bold;
+  text-decoration: underline;
+  background-color: #ec5242;
+  border: 2px solid #fff;
+  padding: 1%;
+}
+
+.logo {
+  display: flex;
+  flex-direction: column;
+  font-family: 'Lato', sans-serif;
+}
+
+.logo-title {
+  align-self: center;
+  border-bottom: 1px solid #ec5242;
+  padding-bottom: 3%;
+}
+
+.logo-description {
+  font-size: 15px;
+  margin-left: 10%;
+  margin-right: 10%;
+  text-align: justify;
+  line-height: 175%;
+}
+
+.logo-image {
+  align-self: center;
+  width: 75%;
+  height: 75%;
+  margin-bottom: 10%;
+}
+
+.partnerships {
+  border-top: 1px solid #d3d3d3;
+  height: 100%;
+  background-color: #272a31;
+  color: #fff;
+  font-family: 'Lato', sans-serif;
+}
+
+.partnerships-title {
+  display: flex;
+  justify-content: center;
+}
+
+.downbeat {
+  width: 20%;
+  height: 30%;
+  margin-left: 15%;
+  margin-bottom: 10%;
+}
+
+.montreux {
+  width: 15%;
+  height: 25%;
+  margin-top: 5%;
+  margin-left: 35%;
+  margin-bottom: 10%;
+}
+
+@media screen and (min-width: 768px) {
+  .social-navbar {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    background-color: #272a31;
+    height: 50px;
+    font-family: 'Lato', sans-serif;
+  }
+
+  .social-media-list {
+    display: flex;
+    flex-direction: row;
+    margin-right: 15%;
+    gap: 10%;
+    list-style: none;
+  }
+
+  a {
+    display: flex;
+    text-decoration: none;
+    color: #d3d3d3;
+  }
+
+  .navbar {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    font-family: 'Lato', sans-serif;
+    font-size: 20px;
+    color: #d3d3d3;
+  }
+
+  .bi-list {
+    display: none;
+  }
+
+  .desktop-title {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .desktop-icon-link {
+    margin-left: 5%;
+    margin-top: 3%;
+    margin-bottom: 3%;
+  }
+
+  .desktop-icon {
+    width: 60%;
+    height: auto;
+    margin-right: 10%;
+    padding-top: 3%;
+    padding-bottom: 3%;
+  }
+
+  .desktop-name {
+    margin-right: 15%;
+    font-family: 'Cocogoose', sans-serif;
+    font-size: 30px;
+  }
+
+  .navbar-items {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+    align-items: center;
+    list-style: none;
+    gap: 30%;
+  }
+
+  .navbar-item {
+    margin-left: -30%;
+  }
+
+  .lede {
+    display: flex;
+    margin-bottom: 0;
+    justify-content: center;
+  }
+
+  .title {
+    display: flex;
+    justify-content: center;
+  }
+
+  .main-description {
+    margin-left: 10%;
+    margin-bottom: 5%;
+    width: 75%;
+    text-align: center;
+  }
+
+  .main-tag {
+    margin-bottom: 0;
+    text-align: center;
+  }
+
+  .email-address {
+    display: flex;
+    justify-content: center;
+  }
+
+  .logo-description {
+    text-align: center;
+  }
+}
```

Comment:
- [ ] Please make sure the CSS styling is built as **mobile-first**, media queries are used to change the design from mobile to desktop, not the other way around.🙏🏽